tritium-tracing/src/main/java/com/palantir/tritium/tracing/TracingInvocationEventHandler.java

```diff
@@ -34,17 +35,36 @@
     private static final Logger logger = LoggerFactory.getLogger(TracingInvocationEventHandler.class);
 
     private final String component;
+    private final boolean shouldUseJavaTracing;
+
+    private static boolean shouldUseJavaTracing() {
+        // ugly but avoids double traces if old http-remoting3 tracing does not delegate to java-tracing
+        Runnable wrappedTrace = com.palantir.remoting3.tracing.Tracers.wrap(() -> {
```

Comment:
Ideally, I'd prefer not to require this class on the classpath.  What about using reflection to just test for the existence of this class, then use .getPackage().getImplementationVersion()?

Reply:
Yeah, I can adjust this to test using reflection and fallback to reflection for slow path when old remoting3 is detected. I won’t keep that in tritium long term, but know there are some places that won’t get both excavated together, so want some transition time.

Reply:
Do you have a time frame you need this change & release?